client/payments/data/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const STORE_KEY = 'wc/payments';
```

Comment:
I think this has a conflict with the WCPay.  Maybe `wc/payment-gateways`?

Reply:
Ah..that makes sense. Updated in https://github.com/woocommerce/woocommerce-admin/pull/6918/commits/fc03047a533aa905596eeeb2452e62ac6b3e9c26